src/qt/sendcoinsdialog.cpp

```diff
@@ -177,10 +199,10 @@ void SendCoinsDialog::setModel(WalletModel *_model)
 
         // set the smartfee-sliders default value (wallets default conf.target or last stored value)
         QSettings settings;
-        if (settings.value("nSmartFeeSliderPosition").toInt() == 0)
-            ui->sliderSmartFee->setValue(ui->sliderSmartFee->maximum() - model->getDefaultConfirmTarget() + 2);
```

Comment:
Hmm... I think an addition is required: we probably need to migrate the old slider value to the new dropdown?